toolbar-ui-angular/README.md

```diff
@@ -0,0 +1,50 @@
+# Glue42 FINOS FDC3 AppD demo toolbar
+
+## Overview
+
+This Project, an open source contribution from Glue42, provides a UI that:
+* connects to one or more FINOS FDC3 Application Directories.
+* shows the applications defined in each server
+* allows those applications to be launched.
+
+The UI is oriented towards developers who are investigating the AppD service, for example it provides easy access to view the JSon definitions of the applications and to easily view the log files. However the UI is functional and a production oriented toolbar could easily be created by removing access to some features.
+
+The UI shows the integration of the FDC3 AppD Services API and the FDC3 API used to launch applications.
+
```

Comment:
It seems a lot of the information above is relevant to the repository as a whole and not just limited to "toolbar-ui-angular" and its README.

I am slightly confused what the relationship is between the "app" component and the "toolbar-ui-angular" component and how they work together.

I think this should be clarified in the root README for this project:
* Overall context
* What components are included
* How do they relate to each other
* How do you run all the components + any dependencies and make them work together

Reply:
README.md is updated.

> # App Directory Demo Launcher aka Toolbar

This Toolbar, contributed by Tick42, connects to one or more FINOS FDC3 compatible application directories that implement the FDC3 App Directory REST API https://fdc3.finos.org/appd-spec. The Toolbar will show all the defined applications, and will attempt to launch applications that are defined by a supported Manifest type.

The Toolbar can also show a set of favourite applications as Icons in the main Toolbar. 

The Toolbar is primarily aimed at developers working with FINOS FDC3, it provides easy access to the application definitions and to the log files to allow developers to explore the services.

## Toolbar components 

The Toolbar is separated into two main components

### app 
The app component is a standalone Electron application. It also includes wrappers for the App Directory services.
It loads the Toolbar window with a URI that is stored in ```./toolbar-ui-angular/dist```. 

### toolbar-ui-angular
The toolbar-ui-angular component is the Toolbar window displayed by the Electron application. It is written with Angular 7. 
By default the Electron application starts the Toolbar window in file mode.

The Toolbar is currently aware of the following manifest types:
- 'org.finos.fdc3.demo' - covers browser/exe and therefore is platform/host independent
- 'org.finos.fdc3.demo.host' - covers platform/host specific applications